src/ts/Offers/Card.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Link } from 'react-router';
 import { FontIcon } from 'material-ui';
 
-export default function CardOwn(props) {
+export default function Card(props) {
```

Comment:
perhaps `Offer` is a less generic name? Worried about possible conflicts with M-UI